.NET/Microsoft.Recognizers.Text.DateTime/Microsoft.Recognizers.Text.DateTime.nuspec

```diff
@@ -18,8 +18,12 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="..\build\package\Microsoft.Recognizers.Text.DateTime.dll" target="lib\net452"/>
-    <file src="..\build\package\Microsoft.Recognizers.Text.DateTime.pdb" target="lib\net452" />
+    <file src="..\build\package\net45\Microsoft.Recognizers.Text.DateTime.dll" target="lib\net45"/>
+    <file src="..\build\package\net45\Microsoft.Recognizers.Text.DateTime.pdb" target="lib\net45" />

```

Comment:
Let's remove the pdbs from the release packages. In the near future we'll publish symbol packages with them